bootstrap.sh

```diff
@@ -344,46 +360,13 @@ install_sentry_cli() {
     logk
     eval "$(sentry-cli bash-hook)"
   fi
-  # Defining it outside of the block ensures that we get reports for 2nd time executions
-  export SENTRY_DSN=https://b70e44882d494c68a78ea1e51c2b17f0@o1.ingest.sentry.io/5480435
```

Comment:
I want to export this *only* when we're not running within the CI, otherwise, I would get spammed in Slack.